canonical/operator
# Define all dependences in requirements files

Defines all project dependencies in one place (`requirements.txt` and `requirements-dev.txt`) and pins them to a version.

Initial review: https://github.com/tinvaan/operator/pull/1

## QA steps

Run the `tox` script and real pebble tests to verify sanity.

```sh
$ tox -r
$ tox -re static
$ tox -re pebble
```
